Library/Formula/portmidi.rb

```diff
@@ -36,6 +41,11 @@ def install
     system "make", "-f", "pm_mac/Makefile.osx", "install"
 
     if build.with? "python"
+      resource("Cython").stage do
+        ENV.prepend_create_path "PYTHONPATH", buildpath+"lib/python2.7/site-packages"
+        system "python", "setup.py", "build", "install", "--prefix=#{buildpath}",
```

Comment:
Can `Language::Python.setup_install_args` be used here? (See [`ansible.rb`](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/ansible.rb) for an example.)